src/renderers/webgl/WebGLTextures.js

```diff
@@ -1138,6 +1182,7 @@ function WebGLTextures( _gl, extensions, state, properties, capabilities, utils,
 		}
 
 		textureProperties.__version = texture.version;
+		return true;
```

Comment:
is this return needed?

Reply:
yes, on line 493 it's used to help determine whether or not to defer texture uploads